falcon_kit/stats_preassembly.py

```diff
@@ -198,6 +192,7 @@ def calc_dict(
         length_cutoff,
     ):
     frag = metric_fragmentation('0-rawreads/preads')
+    trunc = metric_truncation(i_raw_reads_db_fn, '0-rawreads/preads')
```

Comment:
try/catch guard here to catch potential KeyError exception from average_difference method?  Otherwise, assembly may fail at pread stage just because of a report.